src/CheckoutAuthCodeGrant/Views/Shared/_Errors.cshtml

```diff
@@ -6,7 +6,7 @@
     {
         <div class="alert alert-danger" role="alert">
             <strong>@Html.ValueFor(m => error.Error)</strong>
-            <p>@Html.ValueFor(m => error.Description)</p>
+            <p>@Html.Raw(error.Description)</p>
```

Comment:
I'm a little concerned about this change since it opens up the app to an injection attack. You can use this but only if we encode everywhere that we populate `Model.ErrorsViewModel.Errors` with a non-hardcoded value, we encode that value.